source/index.txt

```diff
@@ -95,11 +100,33 @@ What's New
 The :doc:`What's New </whats-new>` section lists new features and
 changes in each version.
 
-Additional Learning
--------------------
+Resources
+---------
+
+Visit the Developer Hub and MongoDB University for resources related to
+the MongoDB Java driver.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub contains content, resources, community, and social
+engagement for developers.
+
+To view articles on Java, see the `How To's and Articles page
+<https://www.mongodb.com/developer/learn/?content=Articles&text=Java>`_.
```

Comment:
Can you use this link instead? It uses the anchor target in the middle of the page where the Java results are rather than the top part of the page which is marketing material:
```suggestion
<https://www.mongodb.com/developer/learn/?content=Articles&text=java#main>`__.
```